src/Exception/ProfilePictureUploadException.php

```diff
@@ -0,0 +1,37 @@
+<?php
+namespace App\Exception;
+
+use Exception;
+
+class ProfilePictureUploadException extends Exception
```

Comment:
I don't see any usage of ProfilePictureUploadException in your code. Add it in registration service and catch it in registration controller. That was the whole point of this ticket. Also make sure to review your own code before me, so we skip this kind of mistakes